src/main/java/appeng/core/sync/GuiBridge.java

```diff
@@ -92,8 +95,11 @@
 import appeng.items.contents.QuartzKnifeObj;
 import appeng.parts.automation.PartFormationPlane;
 import appeng.parts.automation.PartLevelEmitter;
+import appeng.parts.automation.fluid.PartSharedFluidBus;
 import appeng.parts.misc.PartStorageBus;
+import appeng.parts.misc.PartFluidStorageBus;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Wrong order for 'appeng.parts.misc.PartFluidStorageBus' import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)